nsq/event.py

```diff
@@ -28,6 +28,7 @@ class InvalidListenerError(Exception):
 TOUCH = 'touch'
 FINISH = 'finish'
 MESSAGE = 'message'
+MESSAGE_TIMEOUT = 'message_timeout'
```

Comment:
We could shorten this to `TIMEOUT` and `"timeout"` since it's the only timeout event... Would be less descriptive, though.